technique/muon/run_control.py

```diff
@@ -0,0 +1,297 @@
+from __future__ import print_function
+from genie_python import genie as g
+from six.moves import input
+from time import sleep
+
+
+def get_input(prompt):
+    """
+    Get input from the user (allows us to mock it for testing)
```

Comment:
Can we not patch `input` directly? (if not, add justification in the comment)

Reply:
We cannot patch with `test_WHEN_end_pre_cmd_run_information_incorrect_twice_THEN_set_label_called_twice` as it requires a more complex side effect than patching will allow us to do. Added to comment.